src/commands/misc/botinfo.js

```diff
@@ -0,0 +1,31 @@
+const { Command, SwitchbladeEmbed, Constants } = require('../../')
+const Discord = require('discord.js')
+const moment = require('moment')
+
+module.exports = class BotInfo extends Command {
+  constructor (client) {
+    super(client)
+    this.name = 'botinfo'
+  }
+
+  run ({ channel, author, t }) {
+    const embed = new SwitchbladeEmbed(author)
+    const uptime = moment.duration(process.uptime() * 1000).format('d[d] h[h] m[m] s[s]')
+    channel.startTyping()
+
+    embed.setAuthor(this.client.user.username, this.client.user.displayAvatarURL)
+      .setThumbnail(this.client.user.displayAvatarURL)
+      .setDescription([
+        t('commands:botinfo.hello', { this: this }),
+        t('commands:botinfo.statistics', { this: this, uptime, Discord, process })
```

Comment:
Same as above, but here the problem is the `process` variable.

Reply:
keep on reviewing